src/js/utils/deviceFeaturesUtils.js

```diff
@@ -106,3 +108,33 @@ export const getUnavailableCapabilities = (
     });
     return list;
 };
+
+/**
+ * In current versions of FW, TT uses hex raw bytes and T1 uses UTF-8 representation of revision.
+ * Examples:
+ * T1: df0963ec48f01f3d07ffca556e21ff0070cab099, f4424ece1ccb7fc0d6cad00ff840fac287a34f07
+ * TT: 6466303936336563, 623139636266363763
```

Comment:
```suggestion
 * TT: 6466303936336563, 623139636266363763
 * T1: df0963ec48f01f3d07ffca556e21ff0070cab099, f4424ece1ccb7fc0d6cad00ff840fac287a34f07
```

let's preserve relative ordering from the previous sentence. It makes the reading a bit easier.